resource/get_language_list.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+from language_dict import SUPPORTED_LANGUAGES_SPACY
+
+
+def do(payload, config, plugin_config, inputs):
+    language_choices = [{"value": k, "label": v} for k, v in SUPPORTED_LANGUAGES_SPACY.items()]
+    language_choices.insert(0, {"label": "Detected language column", "value": "language_column"})
```

Comment:
The list should be sorted, cf. https://github.com/dataiku/dss-plugin-nlp-preparation/blob/f617292ea3a00322317f9a266c8b8e7fd8e2e242/resource/get_language_list.py#L7